### ISSUE TITLE
Can Leaflet.Photo work with Leaflet.js v. 1.3.1?

### ISSUE THREAD

**Issue by jonberthet** [turban/Leaflet.Photo#9]
Hello,

Great program. Thanks. Is there a version of this app that works with the latest 1.3.1 version of Leaflet.js?

Thanks,

**Comment by theel0ja** [comment]
Yes ([tried myself](https://github.com/theel0ja/view-on-map/tree/36a27cd18cf1e21ddf352adc500653f5042bfae3), but you'll have to update Leaflet.Marketcluster to latest version)
